src/main/resources/db/changelog/db.changelog-RDM-8463.xml

```diff
@@ -0,0 +1,25 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="rdm-8463" author="suman.sreenivasan@hmcts.net">
+        <sql dbms="postgresql"
+             endDelimiter="\nGO"
+             splitStatements="true"
+             stripComments="true">
+            CREATE INDEX IF NOT EXISTS idx_case_data_d8petitionerfirstname ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{D8PetitionerFirstName}'))));
```

Comment:
You can simply specify "CREATE INDEX" only as these  new indexes did not exist in any environment.  But I guess, there is no harm doing a pre-check :)